app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml

```diff
@@ -7,25 +7,29 @@
 /** @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Files */
 
 $_width  = $block->getImagesWidth();
-$_height = $block->getImagesHeight();
 
 ?>
 <?php if ($block->getFilesCount() > 0) : ?>
     <?php foreach ($block->getFiles() as $file) : ?>
+<?php
+$src = $block->escapeHtmlAttr($block->getFileThumbUrl($file));
+$width = $block->escapeHtml($block->getFileWidth($file));
+$height = $block->escapeHtml($block->getFileHeight($file));
+?>
     <div
         data-row="file"
         class="filecnt"
         id="<?= $block->escapeHtmlAttr($block->getFileId($file)) ?>"
         data-size="<?= $block->escapeHtmlAttr($file->getSize()) ?>"
         data-mime-type="<?= $block->escapeHtmlAttr($file->getMimeType()) ?>"
     >
-        <p class="nm" style="height:<?= $block->escapeHtmlAttr($_height) ?>px;">
+        <p class="nm" style="min-height: 72px;">
```

Comment:
Can you please avoid the inline css

Reply:
ok :+1:

Reply:
We can also use `height` attribute if there is no other option.